nntrainer/models/neuralnet.h

```diff
@@ -318,16 +318,6 @@ class NeuralNetwork : public ml::train::Model {
    */
   int addLayer(NodeType layer);
 
-  /**
-   * @brief     join passed graph into the existing graph model
-   * @param[in] graph graph to be added/to extend
-   * @param[in] prefix prefix added to names of layers from this graph
-   * @note It is assumed that this model is valid by itself
-   * @retval #ML_ERROR_NONE Successful.
-   * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
-   */
-  int extendGraph(GraphType graph, std::string prefix = "");
```

Comment:
I thought it is used to extend or merge backbone network into the current network.

Reply:
`addWithReferenceLayers` now takes the job so removed to avoid unmaintained code being present.